test/Motion-test.js

```diff
@@ -1,5 +1,5 @@
+/* eslint-disable class-methods-use-this */
```

Comment:
"fixed" =)

Apparently we can make those methods static according to http://eslint.org/docs/rules/class-methods-use-this

But that is OK for the moment. I'd rather get tests working now.